src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -82,11 +85,18 @@ export class AddBulkConcepts extends Component {
     const lastPage = Math.ceil(cielConcepts.length / conceptsPerPage);
     const lastConcept = indexOfFirstConcept + currentConcepts.length;
     const firstConcept = indexOfLastConcept - 19;
+
+    const disableButton = ((this.state.cielConcepts.length === 0) || this.props.isFetching);
```

Comment:
@ozezzy, I think it is best to destructure all props and states;

Reply:
I have destructured `props` for `isFetching` i'm unable to do that with `cielConcepts` as there are two declarations of it, one from props and the other in state. The one in props has already been destructured in the context